pyproject.toml

```diff
@@ -30,7 +30,7 @@ include = ["eyecite/py.typed"]
 
 [tool.poetry.dependencies]
 python = "^3.7"
-reporters-db = "^3.2.2"
+reporters-db = "3.2.2"
```

Comment:
This was added for testing purposes and will be reverted to the main when complete.